src/Serilog/Context/ImmutableStack.cs

```diff
@@ -15,22 +15,34 @@
 #if LOGCONTEXT
```

Comment:
Ha ha. You're right. I'll see how far I can get on my Mac tonight, otherwise it will be tomorrow morning. I've been meaning to see how the cross-platform bits are working.